### DISCUSSION TITLE
Welcome to ng-material-multilevel-menu Discussions!

### DISCUSSION THREAD

**Discussion by ShankyTiwari** [ShankyTiwari/ng-material-multilevel-menu#168] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by xsala** [comment]
Why to I get that text saying KEY on the right side of the menu options? it appears too in your site example .
thanks

**Reply by ShankyTiwari** [reply]
What do you mean by KEY, can you provide a Screenshot?
